hathor/p2p/manager.py

```diff
@@ -143,6 +149,7 @@ def __init__(self, reactor: Reactor, my_peer: PeerId, server_factory: 'HathorSer
 
     def start(self) -> None:
         self.lc_reconnect.start(5, now=False)
+        self.lc_sync_update.start(self.SYNC_UPDATE_INTERVAL, now=False)
```

Comment:
If all sync managers start with `sync_enabled = False`, does this means that the first 10 minutes you won't be connected to anyone?

Reply:
We enable the sync for all `MAX_ENABLED_SYNC` first connections.

Reply:
Closing this conversation. Thanks!